R/ReadUtils.R

```diff
@@ -40,7 +40,11 @@ read_table_encv1 <- function(dataset, set_index = TRUE) {
   columns <- columns[columns != "__categories"]
 
   col_list <- lapply(columns, function(name) {
-    values <- dataset[[name]]$read()
+    if (dataset[[name]]$dims == 0) {
```

Comment:
perhaps add a comment referencing https://github.com/hhoeflin/hdf5r/issues/118 so we know what this workaround is for (and can remove it in the future)